certs/dodcerts.sh

```diff
@@ -0,0 +1,5 @@
+yum install -y unzip
```

Comment:
leave a comment in the readme and in the shell scripts to say that the scripts should only be run by the container not on the host machine - it's on you to figure out certs if you do a baremetal install of heimdall

Reply:
done